internal/services/storage/storage_share_file_resource.go

```diff
@@ -84,6 +85,12 @@ func resourceStorageShareFile() *pluginsdk.Resource {
 				ForceNew:     true,
 			},
 
+			"content_length": {
```

Comment:
could we add this to the docs?

Reply:
due to its a computed schema, I will add it as a `Attributes Reference` here: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/storage_share_file#attributes-reference